content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -45,24 +42,28 @@ on **one** Kubernetes cluster.
 
 ## Install the Istio remote on every remote cluster
 
-The istio-remote component must be deployed to each remote Kubernetes
-cluster.  There are two approaches to installing the remote.  The remote
-can be installed and managed entirely by Helm and Tiller, or via Helm and
-`kubectl`.
+You must deploy the `istio-remote` component to each remote Kubernetes
+cluster. You can install the component in one of two ways:
 
-### Set environment variables for Pod IPs from Istio control plane needed by remote
+* [Option 1: Use Helm and `kubectl` to install and manage the remote cluster](#kubectl)
 
-Please wait for the Istio control plane to finish initializing
-before proceeding to steps in this section.
+* [Option 2: Use Helm and Tiller to install and manage the remote cluster](#tiller)
 
-These operations must be run on the Istio control plane cluster
-to capture the Istio control-plane service endpoints--e.g. Pilot, Policy,
-and Statsd Pod IP endpoints.
+### Set the needed environment variables {#ip-env-var}
```

Comment:
We don't generally specify id values for our headers. The infrastructure automatically creates headers for every header, and automatically creates linkable targets when you hover over a header in the generated content.

Reply:
I know. :wink: 
However, the generated targets use the full text of the headings and are dependent on that text. Thus, using the generated targets is problematic from an editor's POV. Any change to the heading, breaks all the cross-references to said heading without warning. I introduced the heading labels to solve this problem. Using the ID's, we can edit and change the headings without breaking the cross-references.